torchaudio/functional.py

```diff
@@ -0,0 +1,182 @@
+import numpy as np
```

Comment:
We shouldn't have to depend on numpy. Let's remove this in a future pullrequest.